plugins/home/src/extensions.tsx

```diff
@@ -57,11 +80,23 @@ export function createCardExtension<T>(options: {
   title: string;
   components: () => Promise<ComponentParts>;
   name?: string;
+  data?: Record<string, any>;
+}) {
+  return createCardExtensionInternal<T>(options);
+}
+
+function createCardExtensionInternal<T>(options: {
+  title: string;
+  components: () => Promise<ComponentParts>;
+  name?: string;
+  data?: Record<string, any>;
+  customizableExtension?: boolean;
```

Comment:
The difference is that if the component has 'Settings' defined, it will not be rendered when using customizableCardExtension. This is mainly to maintain some sort of backwards compatibility if people have created card extensions and are using the static home page. 

True that the functionality could be split up, it's quite messy. But didn't have that in the scope of this PR.